pyuvdata/uvdata/uvdata.py

```diff
@@ -23,6 +23,9 @@
 from ..uvbase import UVBase
 
 __all__ = ["UVData"]
+import logging
+
+logger = logging.getLogger(__name__)
```

Comment:
Do we need to keep all these logging lines in?

Reply:
No. I like having some logging -- and I think especially at the INFO/DEBUG level it's not hurting anyone, because it's not printed by default anyway. But some of these are probably egregious (put in because I was looking for something particular). I've taken out all the ones I think are unnecessary. Happy to hear your opinion on others...